### DISCUSSION TITLE
Login Issue on github

### DISCUSSION THREAD

**Discussion by Mahendra2577393** [community/community#140127] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

I forgot my recovery code and I am unable to login into my account there is no any other option how can i retrieve my account.
Give me suggestion.

**Comment by ThisIs-Developer** [comment]
hey @Mahendra2577393,

sorry you’re locked out! try these steps:

1. **check for alternative 2FA methods**: like SMS or an authentication app.
2. **contact GitHub support**: visit their [support page](https://support.github.com/contact) for help.

hope you regain access soon!

---
if this helps, mark it as the answer! 😪
